products/opensuse/main.pm

```diff
@@ -258,7 +258,7 @@ sub load_default_tests {
 }
 
 # load the tests in the right order
-if (is_jeos && !is_container_test) {
+if (is_jeos) {
```

Comment:
This would break the `containers_*` tests that are actually not JeOS but have flavor `JeOS-for-kvm-and-xen`

Reply:
my proposal is actually only having this change :)

Reply:
@mloviska if this is one of the test cases that you had in mind, I run a VR. i cant see how the implementation affects it.

Reply:
Please compare https://openqa.opensuse.org/tests/2031118# vs https://openqa.opensuse.org/tests/2031324#

Reply:
i did an effort to fix this with a dirty approach. I dont know. maybe @jlausuch [suggestion](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13672#issuecomment-968338314) can clean this mess.

Reply:
*this change is ready for review* Still i can remove the duplication of the new block but feel free to go on(because the urgency) if you want and we will fix in another PR if it is needed.